### PR TITLE
CHIA-619 Annotate test_trades.py

### PR DESCRIPTION
**Pull request by AmineKhaldi**

### CODE REVIEW

File: tests/wallet/cat_wallet/test_trades.py [hunk 1]
```diff
@@ -231,61 +231,39 @@ async def test_cat_trades(
...
 
-        proofs_maker: VCProofs = VCProofs({"foo": "1", "bar": "1", "zap": "1"})
-        proof_root_maker: bytes32 = proofs_maker.root()
+        proofs_maker = VCProofs({"foo": "1", "bar": "1", "zap": "1"})
+        proof_root_maker = proofs_maker.root()
         await client_maker.vc_spend(
-            vc_record_maker.vc.launcher_id,
-            wallet_environments.tx_config,
-            new_proof_hash=proof_root_maker,
+            vc_record_maker.vc.launcher_id, wallet_environments.tx_config, new_proof_hash=proof_root_maker
```
Comment (arvidn) [inline tests/wallet/cat_wallet/test_trades.py:249]: is this formatting done automatically by something?
these formatting changes dilute the relevant changes

File: tests/wallet/cat_wallet/test_trades.py [hunk 2]
```diff
@@ -329,20 +307,14 @@ async def test_cat_trades(
...
                 WalletStateTransition(
                     pre_block_balance_updates={
                         "xch": {"set_remainder": True},
                         "new cat": {"init": True, "set_remainder": True},
                     },
-                    post_block_balance_updates={
-                        "xch": {"set_remainder": True},
-                        "new cat": {"set_remainder": True},
-                    },
+                    post_block_balance_updates={"xch": {"set_remainder": True}, "new cat": {"set_remainder": True}},
```
Comment (arvidn) [inline tests/wallet/cat_wallet/test_trades.py:317]: it's not obvious to me that a line that's so long github has to wrap it is better than two lines

File: tests/wallet/cat_wallet/test_trades.py [hunk 3]
```diff
@@ -422,11 +393,8 @@ async def test_cat_trades(
 
     driver_dict: Dict[bytes32, PuzzleInfo] = {}
     for wallet in (cat_wallet_maker, new_cat_wallet_maker):
-        asset_id: str = wallet.get_asset_id()
```
Comment (arvidn) [inline tests/wallet/cat_wallet/test_trades.py:425]: was there a warning about this type annotation?
it has the effect that it protects against `get_asset_id()`'s return value changing

**Review by arvidn — COMMENTED** [review]
this patch is almost entirely formatting changes. I don't think it's a good idea to mix formatting changes and functional changes. It's also not obvious (to me) that these formatting changes are for the better.

File: tests/wallet/cat_wallet/test_trades.py (hunk 2 above)
Comment (Quexington) [reply tests/wallet/cat_wallet/test_trades.py:317]: github doesn't wrap this for me and we have our existing maximum line width enforcement by black so this clearly fits under that limit.

**Review by Quexington — DISMISSED** [review]

File: chia/_tests/wallet/cat_wallet/test_trades.py [hunk 4]
```diff
@@ -1700,19 +1702,18 @@ async def test_trade_cancellation(wallets_prefarm):
...
-async def test_trade_cancellation_balance_check(wallets_prefarm):
-    (
-        [wallet_node_maker, maker_funds],
-        [wallet_node_taker, taker_funds],
-        full_node,
-    ) = wallets_prefarm
+async def test_trade_cancellation_balance_check(
+    wallets_prefarm: Tuple[Tuple[WalletNode, int], Tuple[WalletNode, int], FullNodeSimulator]
+) -> None:
+    (wallet_node_maker, maker_funds), _, full_node = wallets_prefarm
```
Comment (arvidn) [inline chia/_tests/wallet/cat_wallet/test_trades.py:1716]: this used to be destructured from a list, now it's destructured from a tuple. How come this still works?
Does destructuring not care so much about the time as long as it's iterable?

**Review by arvidn — APPROVED** [review]

**Pull request merged**